tests/scripts/optdata/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
That's one small part of it--if that was all I would have probably just written cmd/sh scripts. The bigger part is building up and printing the path(s) where optdata (PGO and/or IBC) needs to get copied for it to be consumed. This is harder than the project.json itself, and figuring it out was much easier in python than a shell script.